experiment/measurer/test_coverage_utils.py

```diff
@@ -14,10 +14,20 @@
 """Tests for coverage_utils.py"""
 import os
 
+import pandas as pd
+
 from experiment.measurer import coverage_utils
 
 TEST_DATA_PATH = os.path.join(os.path.dirname(__file__), 'test_data')
 
+# Expected Constants.
+NUM_FUNCTION_IN_COV_SUMMARY = 3
+NUM_UNRECORDED_SEGMENTS_IN_COV_SUMMARY = 15
+BENCHMARK = 'benchmark_1'
+FUZZER = 'fuzzer_1'
+TIMESTAMP = 900
+TRIAL_ID = 2
```

Comment:
As there is just one test case now, you can move those constants into the test case itself.